orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -1,13 +1,14 @@
-from PyQt4 import QtGui
+from itertools import chain
+
 from PyQt4 import QtCore
-from PyQt4.QtGui import *
+from PyQt4 import QtGui
```

Comment:
`QtGui`is used a few times. About import, I just "organized" them. Isn't that better? Or are you oppose doing this inside an other commit?